protoc-gen-openapiv2/internal/genopenapi/template.go

```diff
@@ -1133,6 +1133,9 @@ func renderServices(services []*descriptor.Service, paths openapiPathsObject, re
 						desc = schema.Description
 						defaultValue = schema.Default
 						extensions = schema.extensions
+						if paramFormat == "" {
+							paramFormat = schema.Format
```

Comment:
Please add a comment here to describe why this is happening
```suggestion
							// If the is no mandatory format based on the field,
							// allow it to be overridden by the user.
							paramFormat = schema.Format
```